geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStoreFactory.scala

```diff
@@ -63,6 +63,7 @@ class HBaseDataStoreFactory extends DataStoreFactorySpi with LazyLogging {
     val coprocessors = CoprocessorConfig(
       enabled = enabledCoprocessors,
       threads = CoprocessorThreadsParam.lookup(params),
+      yieldPartialResults = YieldPartialResultsParam.lookup(params),
```

Comment:
you'll want to add `YieldPartialResultsParam` to the list of advertised params on the HBaseDataStoreFactory object so that it shows up in geoserver

Reply:
Good call.

Reply:
Done.